ui/src/components/filters/QasFilters.vue

```diff
@@ -14,40 +14,24 @@
                 <template #append>
                   <qas-btn v-if="hasSearch" class="q-mr-sm" color="grey-9" icon="sym_r_clear" variant="tertiary" @click="clearSearch" />
 
-                  <qas-btn v-if="useFilterButton" :color="filterButtonColor" data-cy="filters-btn" icon="sym_r_tune" variant="tertiary">
-                    <q-menu anchor="center right" class="full-width" max-width="270px" self="top right">
-                      <div v-if="isFetching" class="q-pa-xl text-center">
-                        <q-spinner color="grey" size="2em" />
-                      </div>
-
-                      <div v-else-if="hasFetchError" class="q-pa-xl text-center">
-                        <q-icon color="negative" name="sym_r_warning" size="2em" />
-                      </div>
-
-                      <q-form v-else class="q-gutter-y-md q-pa-md" @submit.prevent="filter()">
-                        <div v-for="(field, index) in fields" :key="index">
-                          <qas-field v-model="filters[field.name]" :data-cy="`filters-${field.name}-field`" :field="field" v-bind="fieldsProps[field.name]" />
-                        </div>
-
-                        <div class="q-col-gutter-x-md q-mt-xl row">
-                          <div class="col-6">
-                            <qas-btn class="full-width" data-cy="filters-clear-btn" label="Limpar" variant="secondary" @click="clearFilters" />
-                          </div>
-
-                          <div class="col-6">
-                            <qas-btn class="full-width" data-cy="filters-submit-btn" label="Filtrar" type="submit" variant="primary" />
-                          </div>
-                        </div>
-                      </q-form>
-                    </q-menu>
-                  </qas-btn>
+                  <template v-if="showFilterButton">
+                    <slot :context="mx_context" :filter="filter" :filters="activeFilters" name="filter-button" :remove-filter="removeFilter">
+                      <pv-filters-button v-if="useFilterButton" v-model="filters" v-bind="filterButtonProps" />
+                    </slot>
+                  </template>
                 </template>
               </qas-input>
             </div>
           </q-form>
         </slot>
       </div>
 
+      <div v-else-if="showFilterButton" class="col-12 col-md-6">
+        <slot :context="mx_context" :filter="filter" :filters="activeFilters" name="filter-button" :remove-filter="removeFilter">
+          <pv-filters-button v-if="useFilterButton" v-model="filters" v-bind="filterButtonProps" />
+        </slot>
+      </div>
```

Comment:
qual a diferença desse bloco pra o outro? não to conseguindo entender, já q as props e validações são as mesmas

Reply:
Realmente é o mesmo bloco, porém um está dentro do campo de busca para quando a prop `use-search` for `true`, e o outro está fora do campo de search, para quando o `use-search` for `false`.

Por isso foi criado o componente `PvFiltersButton` para que reutilize ele em ambos os locais.